test/basic_lookups.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const test = require('tape');
+
+const LOOKUP_STRINGS = {
```

Comment:
If these are just inversions of the constants, re-write the test to pull the constant array in and run it through the api for the expected result.

Should try to avoid duplicating constants.